compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/DesktopPlatformInput.desktop.kt

```diff
@@ -177,7 +177,7 @@ internal class DesktopTextInputService(private val component: PlatformComponent)
                 return AttributedString(str).iterator
             }
 
-            override fun getTextLocation(offset: TextHitInfo): Rectangle? {
+            override fun getTextLocation(offset: TextHitInfo?): Rectangle? {
```

Comment:
In the description of this method, it says `Params:
offset – the offset within the composed text, if there is composed text; null otherwise`.

It is started to be `null` in JBR 17.0.9-b1087.9.

Does it mean, that there is no composed text at the moment of calling? If there is, then JBR has a regression.

We also need to check, if Chinese/Japanese/Korean work correctly on JBR where we had a crash.

Reply:
I don't really know what composed text is, but in any case we're not actually using that argument. So it's possible that JBR has a regression, but it does affect this fix.